book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
```

Comment:
- [ ] Once more, let's kindly implement the relationship between a book and rentals. It is important to note that a book `has many` rentals hence we would require a method for adding rentals in this Book class